templates/zerver/help/import-your-settings.md

```diff
@@ -0,0 +1,39 @@
+# Import your settings
+
+When you create a Zulip account using an email address already associated with
+an account in another Zulip organization, on Zulip Cloud or the same self-hosted
+Zulip installation, you can import your user settings from an existing account.
+It's a convenient way to preserve the user settings that you've already customized.
+
+!!! tip ""
+
+    Settings that may not apply to all organizations, such as custom profile
+    fields, will not be imported.
+
+The import will include your:
+
+- [Name and avatar](/#settings/profile)
+- [Privacy settings](/#settings/account-and-privacy)
+- [Display settings](/#settings/display-settings)
+- [Notification settings](/#settings/notifications)
+- Hotspots/tutorial state
```

Comment:
I don't think anyone knows what "hotspots" are; we can change it to something like "tutorial completion status".